src/window/window.rs

```diff
@@ -409,6 +410,16 @@ impl Window {
                            mem::transmute(&mut out[0]));
         }
     }
+    
+    /// Get the current screen as an image
+    pub fn snap_image(&self) -> image::ImageBuffer<image::Rgb<u8>, Vec<u8>> {
```

Comment:
I believe it is more idiomatic to import types directly instead, i.e. write `ImageBuffer` here (without `image::`), and add `use image::ImageBuffer;` above.